packages/gatsby-cli/src/create-cli.js

```diff
@@ -346,6 +346,21 @@ module.exports = argv => {
         }
       ),
     })
+    .command({
+      command: `plugin`,
+      desc: `Welcome to the Gatsby source plugin help center!
```

Comment:
Maybe you ought to remove "source" because even though it's more rare, sometimes people build other kinds of plugins? Just realized that right now.

Reply:
I'll remove it !